app/views/suggested_edit/category_index.html.erb

```diff
@@ -15,6 +18,41 @@
   </div>
 <% end %>
 
+<% if params[:show_decided] != '1' %>
+  <h3>All categories</h3>
+  <table>
+    <tr>
+      <th>category</th>
+      <th>pending edits</th>
+    </tr>
+  <% categories.each do |cat| %>
+      <% next if (cat.min_view_trust_level || -1) > (current_user&.trust_level || 0) %>
+      <% sug_edits = SuggestedEdit.where(post: Post.undeleted.where(category: cat), active: true).count %>
+    <tr>
+      <td>
+      <% if cat == current_cat %>
+        <%= cat.name %>
+      <% else %>
+        <%= link_to suggested_edits_queue_url(cat) do %>
+        <%= cat.name %>
+	<% end %>
+      <% end %>
+      </td>
+      <td>
+      <% if cat == current_cat %>
```

Comment:
I couldn't find a way to avoid the repetition here.  Wrapping the link around the pair of table cells did not work.